common/src/main/java/com/linecorp/centraldogma/internal/api/v1/AccessToken.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.internal.api.v1;
+
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AccessToken {
+
+    private static final String BEARER = "Bearer";
+
+    private final String accessToken;
+
+    private final String tokenType = BEARER;
+
+    private final long expiresIn;
+
+    private final String refreshToken;
+
+    //TODO(minwoox) add scope when it needs
```

Comment:
Add scope if needed. ?

Reply:
As a part of the response, there is a scope.
```
scope (optional) If the scope the user granted is identical to the scope the
app requested, this parameter is optional. If the granted scope is different
from the requested scope, such as if the user modified the scope, then
this parameter is required.
```
in [here](https://www.oauth.com/oauth2-servers/access-tokens/access-token-response/)
But I don't think we need it right now, so just left a comment.

Reply:
I actually meant to update the comment to 'Add scope if needed.' rather than asking to add it in this PR. :-D

Reply:
Oops~ got it.~! Thank you~!